### ISSUE TITLE
Backups falling to move/rename.

### ISSUE THREAD

**Issue by CorneliousJD** [tiredofit/docker-db-backup#122]
### Summary

Testing a backup with a backup-now command results in the error

```
2022-04-24.19:23:06 [DEBUG] ** [db-backup] Moving backup to filesystem
mv: can't rename '/tmp/backups/*.': No such file or directory
```

Note, I have also tried providing a separate temp directory for it to work on in my host file system, but that still fails.

### Steps to reproduce

Updated to :latest to fix issue w/ Postgres backup w/ no pass provided and now this issue has started occuring.

### What is the expected *correct* behavior?

It should create the bakckup and move it.

### Relevant logs and/or screenshots

```
sudo docker exec backups backup-now
** Performing Manual Backup
2022-04-24.19:16:36 [DEBUG] ** [db-backup] Looking for existence of DB_NAME environment variable
2022-04-24.19:16:36 [DEBUG] ** [db-backup] Detected Manual Mode
2022-04-24.19:16:36 [DEBUG] ** [db-backup] Backup routines started time: 2022-04-24 19:16:36 EDT
2022-04-24.19:16:36 [DEBUG] ** [db-backup] Databases Found: xxx_production
2022-04-24.19:16:36 [NOTICE] ** [db-backup] Dumping PostgresSQL database: 'xxx_production' and compressing with 'gzip:3' with '2' threads

2022-04-24.19:16:49 [DEBUG] ** [db-backup] DB Backup Exit Code is 0
2022-04-24.19:16:49 [INFO] ** [db-backup] DB Backup of 'pgsql_xxx_production_db_20220424-191636.sql.gz' completed successfully
2022-04-24.19:16:49 [NOTICE] ** [db-backup] Backup of pgsql_xxx_production_db_20220424-191636.sql.gz created with the size of 99013616 bytes
2022-04-24.19:16:49 [DEBUG] ** [db-backup] Moving backup to filesystem
mv: can't rename '/tmp/backups/*.': No such file or directory
2022-04-24.19:16:49 [NOTICE] ** [db-backup] Sending Backup Statistics to Zabbix
Sending failed. Use option -vv for more detailed output.
Sending failed. Use option -vv for more detailed output.
Sending failed. Use option -vv for more detailed output.
Sending failed. Use option -vv for more detailed output.
2022-04-24.19:17:09 [NOTICE] ** [db-backup] Found POST_SCRIPT environment variable. Executing '/scripts/backup_complete.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     2  100     2    0     0      4      0 --:--:-- --:--:-- --:--:--     4
OK2022-04-24.19:17:09 [NOTICE] ** [db-backup] DB Backup for 'xxx_production' time taken: Hours: 0 Minutes: 00 Seconds: 13
2022-04-24.19:17:09 [INFO] ** [db-backup] Backup routines finish time: 2022-04-24 19:17:09 EDT with overall exit code 0
2022-04-24.19:17:09 [NOTICE] ** [db-backup] Backup routines time taken: Hours: 0 Minutes: 00 Seconds: 33
2022-04-24.19:17:09 [INFO] ** [db-backup] Cleaning up old backups
2022-04-24.19:17:10 [DEBUG] ** [db-backup] Exiting due to manual mode
```

### Environment
```
  backups:
    image: tiredofit/db-backup
    container_name: backups
    volumes:
      - /home/xxx/yyy/backups:/backup
      - /home/xxx/yyy/scripts:/scripts
    restart: unless-stopped
    links:
      - db
    environment:
      TZ: 'America/Detroit'
      DB_TYPE: 'pgsql'
      DB_HOST: 'db'
      DB_NAME: 'xxx_production'
      DB_USER: 'postgres'
      EXTRA_OPTS: '--schema=public --blobs'
      COMPRESSION: 'GZ'
      ENABLE_CHECKSUM: 'FALSE'
      DB_DUMP_FREQ: '360'
      DB_DUMP_BEGIN: '+10'
      DB_CLEANUP_TIME: '10080'
      POST_SCRIPT: '/scripts/backup_complete.sh'
```

- Image version / tag:  tiredofit/db-backup:latest
- Host OS: Ubuntu Server 20.04.4 LTS

<details>
<summary>Any logs | docker-compose.yml</summary>
</details>

**Comment by CorneliousJD** [comment]
Verified the same thing is happening when trying to run it via its schedule as well.

**Comment by tiredofit** [comment]
Thanks for the report. Can you pull `tiredofit/db-backup:develop` and get me the same log again? I have a few more statements in there to see whats in the directory before we try to move it.

**Comment by CorneliousJD** [comment]
> Thanks for the report. Can you pull `tiredofit/db-backup:develop` and get me the same log again? I have a few more statements in there to see whats in the directory before we try to move it.

No problem @tiredofit , here you go.

```
2022-04-25.13:08:32 [NOTICE] ** [db-backup] Dumping PostgresSQL database: 'xxx_production' and compressing with 'gzip:3' with '2' threads
2022-04-25.13:08:32 [DEBUG] ** [db-backup] Temp Location: /tmp/backups | Target: pgsql_xxx_production_db_20220425-130832.sql.gz
2022-04-25.13:08:48 [DEBUG] ** [db-backup] DB Backup Exit Code is 0
2022-04-25.13:08:48 [INFO] ** [db-backup] DB Backup of 'pgsql_xxx_production_db_20220425-130832.sql.gz' completed successfully
*********
total 97720
-rw-r--r--    1 root     root     100063839 Apr 25 13:08 pgsql_xxx_production_db_20220425-130832.sql.gz
*********
2022-04-25.13:08:58 [NOTICE] ** [db-backup] Backup of pgsql_xxx_production_db_20220425-130832.sql.gz created with the size of 100063839 bytes
2022-04-25.13:08:58 [DEBUG] ** [db-backup] Moving backup to filesystem
mv: can't rename '/tmp/backups/*.': No such file or directory
```

**Comment by tiredofit** [comment]
OK, I see something related to not taking any MD5 / SHA1 Hashes that is throwing the error - but your backups should still be moving..  I've updated the `develop` image again with different output which should trap the move files statement a bit better. LMK.

**Comment by CorneliousJD** [comment]
Ok maybe I was overlooking what was happening, I took the error at face value and checked via SCP for backups and I didn't see it actually creating any new ones yesterday but i may not have refreshed my scp GUI... because I see backups from 4/25/2022 now in my backups folder which means they ran overnight before trying the :develop branch. 

I just reverted back to :latest and tried a manual backup and verified it's properly showing up now after freshing my SCP gui window.

Sorry for the false alarm. I'll wait my 10 minutes to make sure I see automated ones running as well but I think we are good, sorry again!
